testing/python/fixture.py

```diff
@@ -488,6 +489,10 @@ def test_method(self, something):
         assert len(arg2fixturedefs) == 1
         assert arg2fixturedefs["something"][0].argname == "something"
 
+    @pytest.mark.skipif(
+        hasattr(sys, "pypy_version_info"),
+        reason="this method of test doesn't work on pypy",
+    )
     def test_request_garbage(self, testdir):
```

Comment:
Change flaky test_request_garbage to provide more debug information

This test fails *very* rarely when running in xdist.


Not a part of the purpose of this PR, but I didn't feel like making a new PR just for that.